src/GeoJsonMap.js

```diff
@@ -199,38 +265,87 @@ function Main (props) {
                   <Table.Cell>
                     { apnData['apn'] }
                   </Table.Cell>
-                </Table.Row>
-                <Table.Row>
                   <Table.Cell>
-                    Water Agency / Irrigation District
+                    { apnData1['apn'] }
                   </Table.Cell>
                   <Table.Cell>
-                    { apnData['agency_name'] }
+                    { apnData2['apn'] }
+                  </Table.Cell>
+                  <Table.Cell>
+                    { apnData3['apn'] }
                   </Table.Cell>
                 </Table.Row>
+            
                 <Table.Row>
                   <Table.Cell>
-                    Crop Type
+                    Crop
                   </Table.Cell>
                   <Table.Cell>
                     { apnData['crop2016'] }
                   </Table.Cell>
+                  <Table.Cell>
+                    { apnData1['crop2016'] }
+                  </Table.Cell>
+                  <Table.Cell>
+                    { apnData2['crop2016'] }
+                  </Table.Cell>
+                  <Table.Cell>
+                    { apnData3['crop2016'] }
+                  </Table.Cell>
```

Comment:
What happens to the table when we have only one dict being returned for a single APN?
What happens to the table when we have over 4 dicts be ing returned for a single APN?